README.md

```diff
@@ -1 +1,43 @@
-# oak-actions
+# oak-terraform-actions
+
+The `actions` folder is designed to house reusable GitHub Actions, making it easy to manage and share automation across different repositories. Each subfolder within `actions` contains a specific action with all necessary files and documentation.
```

Comment:
Does this sentence still make sense given the change of function?

If this is for multiple terraform checks then the `actions\terraform\` folder should probably be a little less generic in name.

Otherwise I'm assuming this is just for the one Action so this sentence probably needs rewording.

Reply:
Reworded, resolving!